pkg/debuggers/nodejs/impl.go

```diff
@@ -0,0 +1,59 @@
+package nodejs
+
+import (
+	"syscall"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/solo-io/squash/pkg/debuggers"
+)
+
+type NodeJsInterface struct{}
+type NodeJsInterface8 struct{}
+
+type nodejsDebugServer struct {
+	port int
+}
+
+func (g *nodejsDebugServer) Detach() error {
+	return nil
+}
+
+func (g *nodejsDebugServer) Port() int {
+	return g.port
+}
+
+func (g *nodejsDebugServer) HostType() debuggers.DebugHostType {
+	return debuggers.DebugHostTypeTarget
+}
+
+func (g *NodeJsInterface) Attach(pid int) (debuggers.DebugServer, error) {
+	err := enableDebugger(pid)
+	if err != nil {
+		return nil, err
+	}
+	gds := &nodejsDebugServer{
+		port: 5858,
+	}
+	return gds, nil
+}
+
+func (g *NodeJsInterface8) Attach(pid int) (debuggers.DebugServer, error) {
```

Comment:
this is exactly the same as the other attach except for the port, right?
Perhaps we can merge them?
I'm thinking moving the attach to nodejsDebugServer and creating a 'constructor'  NewNodeDebugger(port int).
Adding the ports as constants, and then you can register them as node.NewNodeDebugger(node.Node8Port).
thoughts?